<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="IBM-PF38ZP4L" failures="41" tests="184" name="Test" time="2717.662" errors="0" timestamp="2025-01-10T11:37:43 IST">
  <testcase classname="testCases.EADMF26_28_TC_02" name="Login" time="5.515"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.831">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A grouping of assets that perform one or more processes to deliver the function of the facility. With respect to above ground infrastructure, facilities are usually within a fenced complex.  For linear infrastructure it represents a connected network of assets. Facilities form parts of a Service Chain.] but found [A grouping of assets that serve to contain wastewater flow and enable temporary holding and storage of wastewater flow during peak periods of flow or abnormal operational conditions.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A grouping of assets that perform one or more processes to deliver the function of the facility. With respect to above ground infrastructure, facilities are usually within a fenced complex.  For linear infrastructure it represents a connected network of assets. Facilities form parts of a Service Chain.] but found [A grouping of assets that serve to contain wastewater flow and enable temporary holding and storage of wastewater flow during peak periods of flow or abnormal operational conditions.]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.823"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.764"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.854"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.848"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.945"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.877"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.889"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.802"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.803"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.854"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.916"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.856"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.836"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.829"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.886"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.907"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.079"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.11">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A grouping of assets that are used to collect water from a surface catchment area and hold it before transferring to another type of Facility such as a transfer facility or a treatment facility for a potable business product supply  Scheme.] but found [A grouping of assets that are used to collect water from a surface catchment area and hold it before transferring to another type of Facility such as a transfer facility or a treatment facility for a potable business product supply Scheme.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A grouping of assets that are used to collect water from a surface catchment area and hold it before transferring to another type of Facility such as a transfer facility or a treatment facility for a potable business product supply  Scheme.] but found [A grouping of assets that are used to collect water from a surface catchment area and hold it before transferring to another type of Facility such as a transfer facility or a treatment facility for a potable business product supply Scheme.]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.114"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.021"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.048"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.148"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.128"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.132"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.114"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.187"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.16"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.074"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.906"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.019"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.029"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.017"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.935"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.958"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.017"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.088"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.097"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.907"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.974"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.79"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.915"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.95">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A grouping of assets that work together to dispose of the wastewater effluent product. (i.e.  Evaporation / Infiltration ponds / etc)] but found [A grouping of assets that work together to dispose of the wastewater effluent product. (i.e. Evaporation / Infiltration ponds / etc)]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A grouping of assets that work together to dispose of the wastewater effluent product. (i.e.  Evaporation / Infiltration ponds / etc)] but found [A grouping of assets that work together to dispose of the wastewater effluent product. (i.e. Evaporation / Infiltration ponds / etc)]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.929"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.849"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.876">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A grouping of assets that work together to perform the function monitoring a part of the business product provision,  i.e. a site monitoring the flow in the network away from any other facility.] but found [A grouping of assets that work together to perform the function monitoring a part of the business product provision, i.e. a site monitoring the flow in the network away from any other facility.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A grouping of assets that work together to perform the function monitoring a part of the business product provision,  i.e. a site monitoring the flow in the network away from any other facility.] but found [A grouping of assets that work together to perform the function monitoring a part of the business product provision, i.e. a site monitoring the flow in the network away from any other facility.]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.997"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.936"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.057"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.978"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.114"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.932"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.941"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.027"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.051"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.163">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A grouping of network that operates at collect and convey irrigation run off water back into the scheme.  i.e. excess water that is collected and drained back.] but found [A grouping of network that operates at collect and convey irrigation run off water back into the scheme. i.e. excess water that is collected and drained back.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A grouping of network that operates at collect and convey irrigation run off water back into the scheme.  i.e. excess water that is collected and drained back.] but found [A grouping of network that operates at collect and convey irrigation run off water back into the scheme. i.e. excess water that is collected and drained back.]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.019"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.01"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.017"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.049">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A grouping of network that is a group of assets (pipes, access chambers) that are used to collect the wastewater from customer properties.  Commonly referred to as a wastewater collection zone.] but found [A grouping of network that is a group of assets (pipes, access chambers) that are used to collect the wastewater from customer properties. Commonly referred to as a wastewater collection zone.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A grouping of network that is a group of assets (pipes, access chambers) that are used to collect the wastewater from customer properties.  Commonly referred to as a wastewater collection zone.] but found [A grouping of network that is a group of assets (pipes, access chambers) that are used to collect the wastewater from customer properties. Commonly referred to as a wastewater collection zone.]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.944"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="14.939"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.04"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.269"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.37"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.467"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.78"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.156"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.189"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.08"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.261">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A grouping of network assets that work together to convey irrigation water between two facilities such as from a pumping facility to major water storage facility.  Transfer networks do not normally have customers connected to them (any customer that is connected to a distribution network supply is outside the &amp;quot;standard term and condition of service&amp;quot;).] but found [A grouping of network assets that work together to convey irrigation water between two facilities such as from a pumping facility to major water storage facility. Transfer networks do not normally have customers connected to them (any customer that is connected to a distribution network supply is outside the &amp;quot;standard term and condition of service&amp;quot;).]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A grouping of network assets that work together to convey irrigation water between two facilities such as from a pumping facility to major water storage facility.  Transfer networks do not normally have customers connected to them (any customer that is connected to a distribution network supply is outside the "standard term and condition of service").] but found [A grouping of network assets that work together to convey irrigation water between two facilities such as from a pumping facility to major water storage facility. Transfer networks do not normally have customers connected to them (any customer that is connected to a distribution network supply is outside the "standard term and condition of service").]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.042">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A grouping of network assets that work together to convey reuse water between two facilities such as from a pumping facility to major water storage facility.  Transfer networks do not normally have customers connected to them (any customer that is connected to a distribution network supply is outside the &amp;quot;standard term and condition of service&amp;quot;).] but found [A grouping of network assets that work together to convey reuse water between two facilities such as from a pumping facility to major water storage facility. Transfer networks do not normally have customers connected to them (any customer that is connected to a distribution network supply is outside the &amp;quot;standard term and condition of service&amp;quot;).]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A grouping of network assets that work together to convey reuse water between two facilities such as from a pumping facility to major water storage facility.  Transfer networks do not normally have customers connected to them (any customer that is connected to a distribution network supply is outside the "standard term and condition of service").] but found [A grouping of network assets that work together to convey reuse water between two facilities such as from a pumping facility to major water storage facility. Transfer networks do not normally have customers connected to them (any customer that is connected to a distribution network supply is outside the "standard term and condition of service").]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.17">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A grouping of network assets that work together to convey non potable water between two facilities such as from a non-potable water source facility to major non-potable water storage facility.  Transfer networks do not normally have customers connected to them (any customer that is connected to a distribution network supply is outside the &amp;quot;standard term and condition of service&amp;quot;).] but found [A grouping of network assets that work together to convey non potable water between two facilities such as from a non-potable water source facility to major non-potable water storage facility. Transfer networks do not normally have customers connected to them (any customer that is connected to a distribution network supply is outside the &amp;quot;standard term and condition of service&amp;quot;).]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A grouping of network assets that work together to convey non potable water between two facilities such as from a non-potable water source facility to major non-potable water storage facility.  Transfer networks do not normally have customers connected to them (any customer that is connected to a distribution network supply is outside the "standard term and condition of service").] but found [A grouping of network assets that work together to convey non potable water between two facilities such as from a non-potable water source facility to major non-potable water storage facility. Transfer networks do not normally have customers connected to them (any customer that is connected to a distribution network supply is outside the "standard term and condition of service").]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.158"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.385">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A grouping of network assets that work together to convey potable water between two facilities such as from a potable water source facility to a potable water treatment facility, or from a potable treatment plant to major potable water storage facility.  Transfer networks do not normally have customers connected to them (any customer that is connected to a distribution network supply is outside the &amp;quot;standard term and condition of service&amp;quot;).] but found [A grouping of network assets that work together to convey potable water between two facilities such as from a potable water source facility to a potable water treatment facility, or from a potable treatment plant to major potable water storage facility. Transfer networks do not normally have customers connected to them (any customer that is connected to a distribution network supply is outside the &amp;quot;standard term and condition of service&amp;quot;).]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A grouping of network assets that work together to convey potable water between two facilities such as from a potable water source facility to a potable water treatment facility, or from a potable treatment plant to major potable water storage facility.  Transfer networks do not normally have customers connected to them (any customer that is connected to a distribution network supply is outside the "standard term and condition of service").] but found [A grouping of network assets that work together to convey potable water between two facilities such as from a potable water source facility to a potable water treatment facility, or from a potable treatment plant to major potable water storage facility. Transfer networks do not normally have customers connected to them (any customer that is connected to a distribution network supply is outside the "standard term and condition of service").]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.18"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.12"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.261"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.1"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.255"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.036"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.272"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.138"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.294"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.422"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.294"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.199">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A grouping of assets that receives non-potable water from a Facility and delivers it to another Facility (i.e.  Direct to a water reticulation network) whilst increasing the hydraulic pressure of the delivered water.] but found [A grouping of assets that receives non-potable water from a Facility and delivers it to another Facility (i.e. Direct to a water reticulation network) whilst increasing the hydraulic pressure of the delivered water.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A grouping of assets that receives non-potable water from a Facility and delivers it to another Facility (i.e.  Direct to a water reticulation network) whilst increasing the hydraulic pressure of the delivered water.] but found [A grouping of assets that receives non-potable water from a Facility and delivers it to another Facility (i.e. Direct to a water reticulation network) whilst increasing the hydraulic pressure of the delivered water.]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.387"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.352"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.483"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.279"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.358"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.246">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A grouping of assets that receives non-potable business product and applies kinetic energy to deliver (lift)  the product to another Facility.] but found [A grouping of assets that receives non-potable business product and applies kinetic energy to deliver (lift) the product to another Facility.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A grouping of assets that receives non-potable business product and applies kinetic energy to deliver (lift)  the product to another Facility.] but found [A grouping of assets that receives non-potable business product and applies kinetic energy to deliver (lift) the product to another Facility.]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.59"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.495"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.274"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.299"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.353"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.494"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.352"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.523"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.464"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.437"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.413">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A collection of assets that serve to contain and hold / store the flow of liquid in a structure that is formed naturally or artificial in the ground during periods of high flow  i.e. a rain events where the network is not able to handle all the flow.] but found [A collection of assets that serve to contain and hold / store the flow of liquid in a structure that is formed naturally or artificial in the ground during periods of high flow i.e. a rain events where the network is not able to handle all the flow.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A collection of assets that serve to contain and hold / store the flow of liquid in a structure that is formed naturally or artificial in the ground during periods of high flow  i.e. a rain events where the network is not able to handle all the flow.] but found [A collection of assets that serve to contain and hold / store the flow of liquid in a structure that is formed naturally or artificial in the ground during periods of high flow i.e. a rain events where the network is not able to handle all the flow.]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.508"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.434">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A collection of assets that holds back raw water from a source (collect rain run off flows),  it is a barrier constructed across a water course to impound water for the purpose of distributed to customer as a non-potable supply product.] but found [A collection of assets that holds back raw water from a source (collect rain run off flows), it is a barrier constructed across a water course to impound water for the purpose of distributed to customer as a non-potable supply product.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A collection of assets that holds back raw water from a source (collect rain run off flows),  it is a barrier constructed across a water course to impound water for the purpose of distributed to customer as a non-potable supply product.] but found [A collection of assets that holds back raw water from a source (collect rain run off flows), it is a barrier constructed across a water course to impound water for the purpose of distributed to customer as a non-potable supply product.]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.572">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A collection of assets that holds back raw water from a source (collect rain run off flows),  it is a barrier constructed across a water course to impound water for the purpose of treatment and distributed to customer as irrigation water.] but found [A collection of assets that holds back raw water from a source (collect rain run off flows), it is a barrier constructed across a water course to impound water for the purpose of treatment and distributed to customer as irrigation water.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A collection of assets that holds back raw water from a source (collect rain run off flows),  it is a barrier constructed across a water course to impound water for the purpose of treatment and distributed to customer as irrigation water.] but found [A collection of assets that holds back raw water from a source (collect rain run off flows), it is a barrier constructed across a water course to impound water for the purpose of treatment and distributed to customer as irrigation water.]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.565">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A collection of assets that holds back raw water from a source (collect rain run off flows),  it is a barrier constructed across a water course to impound water for the purpose of treatment and distributed to customer as reuse water.] but found [A collection of assets that holds back raw water from a source (collect rain run off flows), it is a barrier constructed across a water course to impound water for the purpose of treatment and distributed to customer as reuse water.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A collection of assets that holds back raw water from a source (collect rain run off flows),  it is a barrier constructed across a water course to impound water for the purpose of treatment and distributed to customer as reuse water.] but found [A collection of assets that holds back raw water from a source (collect rain run off flows), it is a barrier constructed across a water course to impound water for the purpose of treatment and distributed to customer as reuse water.]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.721">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A collection of assets that holds back raw water from a source (collect rain run off flows),  it is a barrier constructed across a water course to impound water for the purpose of treatment and distributed to customer as non-potable water.] but found [A collection of assets that holds back raw water from a source (collect rain run off flows), it is a barrier constructed across a water course to impound water for the purpose of treatment and distributed to customer as non-potable water.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A collection of assets that holds back raw water from a source (collect rain run off flows),  it is a barrier constructed across a water course to impound water for the purpose of treatment and distributed to customer as non-potable water.] but found [A collection of assets that holds back raw water from a source (collect rain run off flows), it is a barrier constructed across a water course to impound water for the purpose of treatment and distributed to customer as non-potable water.]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.616">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A collection of assets that holds back raw water from a source (collect rain run off flows),  it is a barrier constructed across a water course to impound water for the purpose of distributed to customer as a potable water supply product.] but found [A collection of assets that holds back raw water from a source (collect rain run off flows), it is a barrier constructed across a water course to impound water for the purpose of distributed to customer as a potable water supply product.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A collection of assets that holds back raw water from a source (collect rain run off flows),  it is a barrier constructed across a water course to impound water for the purpose of distributed to customer as a potable water supply product.] but found [A collection of assets that holds back raw water from a source (collect rain run off flows), it is a barrier constructed across a water course to impound water for the purpose of distributed to customer as a potable water supply product.]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.675"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.733">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A collection of assets that holds back raw water from a source (collect rain run off flows),  it is a barrier constructed across a water course to impound water for the purpose of treatment and distributed to customer as potable water.] but found [A collection of assets that holds back raw water from a source (collect rain run off flows), it is a barrier constructed across a water course to impound water for the purpose of treatment and distributed to customer as potable water.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A collection of assets that holds back raw water from a source (collect rain run off flows),  it is a barrier constructed across a water course to impound water for the purpose of treatment and distributed to customer as potable water.] but found [A collection of assets that holds back raw water from a source (collect rain run off flows), it is a barrier constructed across a water course to impound water for the purpose of treatment and distributed to customer as potable water.]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.822"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.589"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.798"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.733"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.929"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.95"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.91"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.907">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A collection of assets that provide reserve and emergency storage for product (water) before it is distributed to customer to ensure not interrupted water supply (i.e. due to a leak on the main) and ensure consistent water pressure.  Type of tanks: Ground Level Tank or Elevated Tank.] but found [A collection of assets that provide reserve and emergency storage for product (water) before it is distributed to customer to ensure not interrupted water supply (i.e. due to a leak on the main) and ensure consistent water pressure. Type of tanks: Ground Level Tank or Elevated Tank.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A collection of assets that provide reserve and emergency storage for product (water) before it is distributed to customer to ensure not interrupted water supply (i.e. due to a leak on the main) and ensure consistent water pressure.  Type of tanks: Ground Level Tank or Elevated Tank.] but found [A collection of assets that provide reserve and emergency storage for product (water) before it is distributed to customer to ensure not interrupted water supply (i.e. due to a leak on the main) and ensure consistent water pressure. Type of tanks: Ground Level Tank or Elevated Tank.]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.857">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A collection of assets that provide reserve and emergency storage for non-potable product (water) before it is distributed to customer to ensure not interrupted water supply (i.e. due to a leak on the main) and ensure consistent water pressure.  Type of tanks: Ground Level Tank or Elevated Tank.] but found [A collection of assets that provide reserve and emergency storage for non-potable product (water) before it is distributed to customer to ensure not interrupted water supply (i.e. due to a leak on the main) and ensure consistent water pressure. Type of tanks: Ground Level Tank or Elevated Tank.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A collection of assets that provide reserve and emergency storage for non-potable product (water) before it is distributed to customer to ensure not interrupted water supply (i.e. due to a leak on the main) and ensure consistent water pressure.  Type of tanks: Ground Level Tank or Elevated Tank.] but found [A collection of assets that provide reserve and emergency storage for non-potable product (water) before it is distributed to customer to ensure not interrupted water supply (i.e. due to a leak on the main) and ensure consistent water pressure. Type of tanks: Ground Level Tank or Elevated Tank.]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.036"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.036">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A collection of assets that provide reserve and emergency storage for reuse water before it is distributed to customer to ensure not interrupted water supply (i.e.. due to a leak on the main) and ensure consistent water pressure.  Type of tanks: Ground Level Tank or Elevated Tank.] but found [A collection of assets that provide reserve and emergency storage for reuse water before it is distributed to customer to ensure not interrupted water supply (i.e.. due to a leak on the main) and ensure consistent water pressure. Type of tanks: Ground Level Tank or Elevated Tank.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A collection of assets that provide reserve and emergency storage for reuse water before it is distributed to customer to ensure not interrupted water supply (i.e.. due to a leak on the main) and ensure consistent water pressure.  Type of tanks: Ground Level Tank or Elevated Tank.] but found [A collection of assets that provide reserve and emergency storage for reuse water before it is distributed to customer to ensure not interrupted water supply (i.e.. due to a leak on the main) and ensure consistent water pressure. Type of tanks: Ground Level Tank or Elevated Tank.]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="15.945">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A collection of assets that provide reserve and emergency storage for non-potable water before it is distributed to customer to ensure not interrupted water supply (i.e.. due to a leak on the main) and ensure consistent water pressure.  Type of tanks: Ground Level Tank or Elevated Tank.] but found [A collection of assets that provide reserve and emergency storage for non-potable water before it is distributed to customer to ensure not interrupted water supply (i.e.. due to a leak on the main) and ensure consistent water pressure. Type of tanks: Ground Level Tank or Elevated Tank.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A collection of assets that provide reserve and emergency storage for non-potable water before it is distributed to customer to ensure not interrupted water supply (i.e.. due to a leak on the main) and ensure consistent water pressure.  Type of tanks: Ground Level Tank or Elevated Tank.] but found [A collection of assets that provide reserve and emergency storage for non-potable water before it is distributed to customer to ensure not interrupted water supply (i.e.. due to a leak on the main) and ensure consistent water pressure. Type of tanks: Ground Level Tank or Elevated Tank.]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.028">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A collection of assets that provide reserve and emergency storage for potable product (water) before it is distributed to customer to ensure not interrupted water supply (i.e. due to a leak on the main) and ensure consistent water pressure.  Type of tanks: Ground Level Tank or Elevated Tank.] but found [A collection of assets that provide reserve and emergency storage for potable product (water) before it is distributed to customer to ensure not interrupted water supply (i.e. due to a leak on the main) and ensure consistent water pressure. Type of tanks: Ground Level Tank or Elevated Tank.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A collection of assets that provide reserve and emergency storage for potable product (water) before it is distributed to customer to ensure not interrupted water supply (i.e. due to a leak on the main) and ensure consistent water pressure.  Type of tanks: Ground Level Tank or Elevated Tank.] but found [A collection of assets that provide reserve and emergency storage for potable product (water) before it is distributed to customer to ensure not interrupted water supply (i.e. due to a leak on the main) and ensure consistent water pressure. Type of tanks: Ground Level Tank or Elevated Tank.]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.037">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [A collection of assets that provide reserve and emergency storage for (treated) potable water before it is distributed to customer to ensure not interrupted water supply (i.e.. due to a leak on the main) and ensure consistent water pressure.  Type of tanks: Ground Level Tank or Elevated Tank.] but found [A collection of assets that provide reserve and emergency storage for (treated) potable water before it is distributed to customer to ensure not interrupted water supply (i.e.. due to a leak on the main) and ensure consistent water pressure. Type of tanks: Ground Level Tank or Elevated Tank.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [A collection of assets that provide reserve and emergency storage for (treated) potable water before it is distributed to customer to ensure not interrupted water supply (i.e.. due to a leak on the main) and ensure consistent water pressure.  Type of tanks: Ground Level Tank or Elevated Tank.] but found [A collection of assets that provide reserve and emergency storage for (treated) potable water before it is distributed to customer to ensure not interrupted water supply (i.e.. due to a leak on the main) and ensure consistent water pressure. Type of tanks: Ground Level Tank or Elevated Tank.]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.107"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.517"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.507"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.278"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.409"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.378"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.6"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.64"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.633"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.599"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.613"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.582"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.598"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.95"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.655"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.962"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.803"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.742"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.959"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.893"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="16.966"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="17.024"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="17.423"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="17.643"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="17.803"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="17.172"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="17.215"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="17.399"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="17.518"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="17.666"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="17.593"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="17.254"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="17.234"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="17.516"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="18.147"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="17.596"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="17.44"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="17.527"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="17.55"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="17.528"/>
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="5.33">
    <failure type="java.lang.AssertionError" message="expected [] but found [EMERGENCY]">
      <![CDATA[java.lang.AssertionError: expected [] but found [EMERGENCY]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="2.118">
    <failure type="org.openqa.selenium.ElementClickInterceptedException" message="element click intercepted: Element &amp;lt;input id=&amp;quot;quicksearchQSMenuImage&amp;quot; type=&amp;quot;image&amp;quot; title=&amp;quot;Quick Search menu&amp;quot; alt=&amp;quot;Quick Search menu&amp;quot; aria-label=&amp;quot;Quick Search menu&amp;quot; role=&amp;quot;button&amp;quot; onkeypress=&amp;quot;if(event.key === &amp;apos;Enter&amp;apos;){event.target.click();}&amp;quot; onclick=&amp;quot;setClickPosition(this, event); sendEvent(&amp;apos;click&amp;apos;, &amp;apos;quicksearch&amp;apos;, &amp;apos;quicksearch&amp;apos;)&amp;quot; class=&amp;quot;greybutton&amp;quot; src=&amp;quot;nav_btn_menuArrow_Search.gif&amp;quot;&amp;gt; is not clickable at point (1274, 36). Other element would receive the click: &amp;lt;div id=&amp;quot;msgbox-dialog_inner_dialogwait&amp;quot; class=&amp;quot;wait_modal&amp;quot; style=&amp;quot;cursor: wait; height: 643px; width: 1584px; position: absolute; left: 0px; top: 0px; display: block;&amp;quot;&amp;gt;&amp;lt;/div&amp;gt;
  (Session info: chrome=131.0.6778.265)
Build info: version: &amp;apos;4.26.0&amp;apos;, revision: &amp;apos;8ccf0219d7&amp;apos;
System info: os.name: &amp;apos;Windows 11&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;23.0.1&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [eb8877bee05955f8915be45a7bba5230, clickElement {id=f.A187692B93ADA8541EA4F2EAA806781B.d.F5A522AA3D1AF5DA3E26501D3085A60F.e.356}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 131.0.6778.265, chrome: {chromedriverVersion: 131.0.6778.264 (2d05e315153..., userDataDir: C:\Users\VENKAT~1\AppData\L...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:49911}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:49911/devtoo..., se:cdpVersion: 131.0.6778.265, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (eb8877bee05955f8915be45a7bba5230)] -&amp;gt; xpath: //input[@title=&amp;apos;Quick Search menu&amp;apos;]]
Session ID: eb8877bee05955f8915be45a7bba5230">
      <![CDATA[org.openqa.selenium.ElementClickInterceptedException: element click intercepted: Element <input id="quicksearchQSMenuImage" type="image" title="Quick Search menu" alt="Quick Search menu" aria-label="Quick Search menu" role="button" onkeypress="if(event.key === 'Enter'){event.target.click();}" onclick="setClickPosition(this, event); sendEvent('click', 'quicksearch', 'quicksearch')" class="greybutton" src="nav_btn_menuArrow_Search.gif"> is not clickable at point (1274, 36). Other element would receive the click: <div id="msgbox-dialog_inner_dialogwait" class="wait_modal" style="cursor: wait; height: 643px; width: 1584px; position: absolute; left: 0px; top: 0px; display: block;"></div>
  (Session info: chrome=131.0.6778.265)
Build info: version: '4.26.0', revision: '8ccf0219d7'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '23.0.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [eb8877bee05955f8915be45a7bba5230, clickElement {id=f.A187692B93ADA8541EA4F2EAA806781B.d.F5A522AA3D1AF5DA3E26501D3085A60F.e.356}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 131.0.6778.265, chrome: {chromedriverVersion: 131.0.6778.264 (2d05e315153..., userDataDir: C:\Users\VENKAT~1\AppData\L...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:49911}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:49911/devtoo..., se:cdpVersion: 131.0.6778.265, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (eb8877bee05955f8915be45a7bba5230)] -> xpath: //input[@title='Quick Search menu']]
Session ID: eb8877bee05955f8915be45a7bba5230
at org.openqa.selenium.remote.ErrorCodec.decode(ErrorCodec.java:167)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:138)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:50)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:190)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:223)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:76)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
at jdk.proxy2/jdk.proxy2.$Proxy28.click(Unknown Source)
at pageObjects.ClassificationsPage.clickQuickSearchMenu(ClassificationsPage.java:80)
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="2.233">
    <failure type="org.openqa.selenium.ElementClickInterceptedException" message="element click intercepted: Element &amp;lt;input id=&amp;quot;quicksearchQSMenuImage&amp;quot; type=&amp;quot;image&amp;quot; title=&amp;quot;Quick Search menu&amp;quot; alt=&amp;quot;Quick Search menu&amp;quot; aria-label=&amp;quot;Quick Search menu&amp;quot; role=&amp;quot;button&amp;quot; onkeypress=&amp;quot;if(event.key === &amp;apos;Enter&amp;apos;){event.target.click();}&amp;quot; onclick=&amp;quot;setClickPosition(this, event); sendEvent(&amp;apos;click&amp;apos;, &amp;apos;quicksearch&amp;apos;, &amp;apos;quicksearch&amp;apos;)&amp;quot; class=&amp;quot;greybutton&amp;quot; src=&amp;quot;nav_btn_menuArrow_Search.gif&amp;quot;&amp;gt; is not clickable at point (1274, 36). Other element would receive the click: &amp;lt;div id=&amp;quot;msgbox-dialog_inner_dialogwait&amp;quot; class=&amp;quot;wait_modal&amp;quot; style=&amp;quot;cursor: wait; height: 643px; width: 1584px; position: absolute; left: 0px; top: 0px; display: block;&amp;quot;&amp;gt;&amp;lt;/div&amp;gt;
  (Session info: chrome=131.0.6778.265)
Build info: version: &amp;apos;4.26.0&amp;apos;, revision: &amp;apos;8ccf0219d7&amp;apos;
System info: os.name: &amp;apos;Windows 11&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;23.0.1&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [eb8877bee05955f8915be45a7bba5230, clickElement {id=f.A187692B93ADA8541EA4F2EAA806781B.d.F5A522AA3D1AF5DA3E26501D3085A60F.e.356}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 131.0.6778.265, chrome: {chromedriverVersion: 131.0.6778.264 (2d05e315153..., userDataDir: C:\Users\VENKAT~1\AppData\L...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:49911}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:49911/devtoo..., se:cdpVersion: 131.0.6778.265, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (eb8877bee05955f8915be45a7bba5230)] -&amp;gt; xpath: //input[@title=&amp;apos;Quick Search menu&amp;apos;]]
Session ID: eb8877bee05955f8915be45a7bba5230">
      <![CDATA[org.openqa.selenium.ElementClickInterceptedException: element click intercepted: Element <input id="quicksearchQSMenuImage" type="image" title="Quick Search menu" alt="Quick Search menu" aria-label="Quick Search menu" role="button" onkeypress="if(event.key === 'Enter'){event.target.click();}" onclick="setClickPosition(this, event); sendEvent('click', 'quicksearch', 'quicksearch')" class="greybutton" src="nav_btn_menuArrow_Search.gif"> is not clickable at point (1274, 36). Other element would receive the click: <div id="msgbox-dialog_inner_dialogwait" class="wait_modal" style="cursor: wait; height: 643px; width: 1584px; position: absolute; left: 0px; top: 0px; display: block;"></div>
  (Session info: chrome=131.0.6778.265)
Build info: version: '4.26.0', revision: '8ccf0219d7'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '23.0.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [eb8877bee05955f8915be45a7bba5230, clickElement {id=f.A187692B93ADA8541EA4F2EAA806781B.d.F5A522AA3D1AF5DA3E26501D3085A60F.e.356}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 131.0.6778.265, chrome: {chromedriverVersion: 131.0.6778.264 (2d05e315153..., userDataDir: C:\Users\VENKAT~1\AppData\L...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:49911}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:49911/devtoo..., se:cdpVersion: 131.0.6778.265, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (eb8877bee05955f8915be45a7bba5230)] -> xpath: //input[@title='Quick Search menu']]
Session ID: eb8877bee05955f8915be45a7bba5230
at org.openqa.selenium.remote.ErrorCodec.decode(ErrorCodec.java:167)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:138)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:50)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:190)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:223)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:76)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
at jdk.proxy2/jdk.proxy2.$Proxy28.click(Unknown Source)
at pageObjects.ClassificationsPage.clickQuickSearchMenu(ClassificationsPage.java:80)
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="2.069">
    <failure type="org.openqa.selenium.ElementClickInterceptedException" message="element click intercepted: Element &amp;lt;input id=&amp;quot;quicksearchQSMenuImage&amp;quot; type=&amp;quot;image&amp;quot; title=&amp;quot;Quick Search menu&amp;quot; alt=&amp;quot;Quick Search menu&amp;quot; aria-label=&amp;quot;Quick Search menu&amp;quot; role=&amp;quot;button&amp;quot; onkeypress=&amp;quot;if(event.key === &amp;apos;Enter&amp;apos;){event.target.click();}&amp;quot; onclick=&amp;quot;setClickPosition(this, event); sendEvent(&amp;apos;click&amp;apos;, &amp;apos;quicksearch&amp;apos;, &amp;apos;quicksearch&amp;apos;)&amp;quot; class=&amp;quot;greybutton&amp;quot; src=&amp;quot;nav_btn_menuArrow_Search.gif&amp;quot;&amp;gt; is not clickable at point (1274, 36). Other element would receive the click: &amp;lt;div id=&amp;quot;msgbox-dialog_inner_dialogwait&amp;quot; class=&amp;quot;wait_modal&amp;quot; style=&amp;quot;cursor: wait; height: 643px; width: 1584px; position: absolute; left: 0px; top: 0px; display: block;&amp;quot;&amp;gt;&amp;lt;/div&amp;gt;
  (Session info: chrome=131.0.6778.265)
Build info: version: &amp;apos;4.26.0&amp;apos;, revision: &amp;apos;8ccf0219d7&amp;apos;
System info: os.name: &amp;apos;Windows 11&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;23.0.1&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [eb8877bee05955f8915be45a7bba5230, clickElement {id=f.A187692B93ADA8541EA4F2EAA806781B.d.F5A522AA3D1AF5DA3E26501D3085A60F.e.356}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 131.0.6778.265, chrome: {chromedriverVersion: 131.0.6778.264 (2d05e315153..., userDataDir: C:\Users\VENKAT~1\AppData\L...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:49911}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:49911/devtoo..., se:cdpVersion: 131.0.6778.265, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (eb8877bee05955f8915be45a7bba5230)] -&amp;gt; xpath: //input[@title=&amp;apos;Quick Search menu&amp;apos;]]
Session ID: eb8877bee05955f8915be45a7bba5230">
      <![CDATA[org.openqa.selenium.ElementClickInterceptedException: element click intercepted: Element <input id="quicksearchQSMenuImage" type="image" title="Quick Search menu" alt="Quick Search menu" aria-label="Quick Search menu" role="button" onkeypress="if(event.key === 'Enter'){event.target.click();}" onclick="setClickPosition(this, event); sendEvent('click', 'quicksearch', 'quicksearch')" class="greybutton" src="nav_btn_menuArrow_Search.gif"> is not clickable at point (1274, 36). Other element would receive the click: <div id="msgbox-dialog_inner_dialogwait" class="wait_modal" style="cursor: wait; height: 643px; width: 1584px; position: absolute; left: 0px; top: 0px; display: block;"></div>
  (Session info: chrome=131.0.6778.265)
Build info: version: '4.26.0', revision: '8ccf0219d7'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '23.0.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [eb8877bee05955f8915be45a7bba5230, clickElement {id=f.A187692B93ADA8541EA4F2EAA806781B.d.F5A522AA3D1AF5DA3E26501D3085A60F.e.356}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 131.0.6778.265, chrome: {chromedriverVersion: 131.0.6778.264 (2d05e315153..., userDataDir: C:\Users\VENKAT~1\AppData\L...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:49911}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:49911/devtoo..., se:cdpVersion: 131.0.6778.265, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (eb8877bee05955f8915be45a7bba5230)] -> xpath: //input[@title='Quick Search menu']]
Session ID: eb8877bee05955f8915be45a7bba5230
at org.openqa.selenium.remote.ErrorCodec.decode(ErrorCodec.java:167)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:138)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:50)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:190)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:223)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:76)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
at jdk.proxy2/jdk.proxy2.$Proxy28.click(Unknown Source)
at pageObjects.ClassificationsPage.clickQuickSearchMenu(ClassificationsPage.java:80)
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="5.345">
    <failure type="java.lang.AssertionError" message="expected [] but found [EMERGENCY]">
      <![CDATA[java.lang.AssertionError: expected [] but found [EMERGENCY]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="2.108">
    <failure type="org.openqa.selenium.ElementClickInterceptedException" message="element click intercepted: Element &amp;lt;input id=&amp;quot;quicksearchQSMenuImage&amp;quot; type=&amp;quot;image&amp;quot; title=&amp;quot;Quick Search menu&amp;quot; alt=&amp;quot;Quick Search menu&amp;quot; aria-label=&amp;quot;Quick Search menu&amp;quot; role=&amp;quot;button&amp;quot; onkeypress=&amp;quot;if(event.key === &amp;apos;Enter&amp;apos;){event.target.click();}&amp;quot; onclick=&amp;quot;setClickPosition(this, event); sendEvent(&amp;apos;click&amp;apos;, &amp;apos;quicksearch&amp;apos;, &amp;apos;quicksearch&amp;apos;)&amp;quot; class=&amp;quot;greybutton&amp;quot; src=&amp;quot;nav_btn_menuArrow_Search.gif&amp;quot;&amp;gt; is not clickable at point (1274, 36). Other element would receive the click: &amp;lt;div id=&amp;quot;msgbox-dialog_inner_dialogwait&amp;quot; class=&amp;quot;wait_modal&amp;quot; style=&amp;quot;cursor: wait; height: 643px; width: 1584px; position: absolute; left: 0px; top: 0px; display: block;&amp;quot;&amp;gt;&amp;lt;/div&amp;gt;
  (Session info: chrome=131.0.6778.265)
Build info: version: &amp;apos;4.26.0&amp;apos;, revision: &amp;apos;8ccf0219d7&amp;apos;
System info: os.name: &amp;apos;Windows 11&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;23.0.1&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [eb8877bee05955f8915be45a7bba5230, clickElement {id=f.A187692B93ADA8541EA4F2EAA806781B.d.F5A522AA3D1AF5DA3E26501D3085A60F.e.356}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 131.0.6778.265, chrome: {chromedriverVersion: 131.0.6778.264 (2d05e315153..., userDataDir: C:\Users\VENKAT~1\AppData\L...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:49911}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:49911/devtoo..., se:cdpVersion: 131.0.6778.265, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (eb8877bee05955f8915be45a7bba5230)] -&amp;gt; xpath: //input[@title=&amp;apos;Quick Search menu&amp;apos;]]
Session ID: eb8877bee05955f8915be45a7bba5230">
      <![CDATA[org.openqa.selenium.ElementClickInterceptedException: element click intercepted: Element <input id="quicksearchQSMenuImage" type="image" title="Quick Search menu" alt="Quick Search menu" aria-label="Quick Search menu" role="button" onkeypress="if(event.key === 'Enter'){event.target.click();}" onclick="setClickPosition(this, event); sendEvent('click', 'quicksearch', 'quicksearch')" class="greybutton" src="nav_btn_menuArrow_Search.gif"> is not clickable at point (1274, 36). Other element would receive the click: <div id="msgbox-dialog_inner_dialogwait" class="wait_modal" style="cursor: wait; height: 643px; width: 1584px; position: absolute; left: 0px; top: 0px; display: block;"></div>
  (Session info: chrome=131.0.6778.265)
Build info: version: '4.26.0', revision: '8ccf0219d7'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '23.0.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [eb8877bee05955f8915be45a7bba5230, clickElement {id=f.A187692B93ADA8541EA4F2EAA806781B.d.F5A522AA3D1AF5DA3E26501D3085A60F.e.356}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 131.0.6778.265, chrome: {chromedriverVersion: 131.0.6778.264 (2d05e315153..., userDataDir: C:\Users\VENKAT~1\AppData\L...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:49911}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:49911/devtoo..., se:cdpVersion: 131.0.6778.265, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (eb8877bee05955f8915be45a7bba5230)] -> xpath: //input[@title='Quick Search menu']]
Session ID: eb8877bee05955f8915be45a7bba5230
at org.openqa.selenium.remote.ErrorCodec.decode(ErrorCodec.java:167)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:138)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:50)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:190)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:223)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:76)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
at jdk.proxy2/jdk.proxy2.$Proxy28.click(Unknown Source)
at pageObjects.ClassificationsPage.clickQuickSearchMenu(ClassificationsPage.java:80)
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="2.066">
    <failure type="org.openqa.selenium.ElementClickInterceptedException" message="element click intercepted: Element &amp;lt;input id=&amp;quot;quicksearchQSMenuImage&amp;quot; type=&amp;quot;image&amp;quot; title=&amp;quot;Quick Search menu&amp;quot; alt=&amp;quot;Quick Search menu&amp;quot; aria-label=&amp;quot;Quick Search menu&amp;quot; role=&amp;quot;button&amp;quot; onkeypress=&amp;quot;if(event.key === &amp;apos;Enter&amp;apos;){event.target.click();}&amp;quot; onclick=&amp;quot;setClickPosition(this, event); sendEvent(&amp;apos;click&amp;apos;, &amp;apos;quicksearch&amp;apos;, &amp;apos;quicksearch&amp;apos;)&amp;quot; class=&amp;quot;greybutton&amp;quot; src=&amp;quot;nav_btn_menuArrow_Search.gif&amp;quot;&amp;gt; is not clickable at point (1274, 36). Other element would receive the click: &amp;lt;div id=&amp;quot;msgbox-dialog_inner_dialogwait&amp;quot; class=&amp;quot;wait_modal&amp;quot; style=&amp;quot;cursor: wait; height: 643px; width: 1584px; position: absolute; left: 0px; top: 0px; display: block;&amp;quot;&amp;gt;&amp;lt;/div&amp;gt;
  (Session info: chrome=131.0.6778.265)
Build info: version: &amp;apos;4.26.0&amp;apos;, revision: &amp;apos;8ccf0219d7&amp;apos;
System info: os.name: &amp;apos;Windows 11&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;23.0.1&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [eb8877bee05955f8915be45a7bba5230, clickElement {id=f.A187692B93ADA8541EA4F2EAA806781B.d.F5A522AA3D1AF5DA3E26501D3085A60F.e.356}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 131.0.6778.265, chrome: {chromedriverVersion: 131.0.6778.264 (2d05e315153..., userDataDir: C:\Users\VENKAT~1\AppData\L...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:49911}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:49911/devtoo..., se:cdpVersion: 131.0.6778.265, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (eb8877bee05955f8915be45a7bba5230)] -&amp;gt; xpath: //input[@title=&amp;apos;Quick Search menu&amp;apos;]]
Session ID: eb8877bee05955f8915be45a7bba5230">
      <![CDATA[org.openqa.selenium.ElementClickInterceptedException: element click intercepted: Element <input id="quicksearchQSMenuImage" type="image" title="Quick Search menu" alt="Quick Search menu" aria-label="Quick Search menu" role="button" onkeypress="if(event.key === 'Enter'){event.target.click();}" onclick="setClickPosition(this, event); sendEvent('click', 'quicksearch', 'quicksearch')" class="greybutton" src="nav_btn_menuArrow_Search.gif"> is not clickable at point (1274, 36). Other element would receive the click: <div id="msgbox-dialog_inner_dialogwait" class="wait_modal" style="cursor: wait; height: 643px; width: 1584px; position: absolute; left: 0px; top: 0px; display: block;"></div>
  (Session info: chrome=131.0.6778.265)
Build info: version: '4.26.0', revision: '8ccf0219d7'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '23.0.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [eb8877bee05955f8915be45a7bba5230, clickElement {id=f.A187692B93ADA8541EA4F2EAA806781B.d.F5A522AA3D1AF5DA3E26501D3085A60F.e.356}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 131.0.6778.265, chrome: {chromedriverVersion: 131.0.6778.264 (2d05e315153..., userDataDir: C:\Users\VENKAT~1\AppData\L...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:49911}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:49911/devtoo..., se:cdpVersion: 131.0.6778.265, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (eb8877bee05955f8915be45a7bba5230)] -> xpath: //input[@title='Quick Search menu']]
Session ID: eb8877bee05955f8915be45a7bba5230
at org.openqa.selenium.remote.ErrorCodec.decode(ErrorCodec.java:167)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:138)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:50)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:190)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:223)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:76)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
at jdk.proxy2/jdk.proxy2.$Proxy28.click(Unknown Source)
at pageObjects.ClassificationsPage.clickQuickSearchMenu(ClassificationsPage.java:80)
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="5.309">
    <failure type="java.lang.AssertionError" message="expected [] but found [EMERGENCY]">
      <![CDATA[java.lang.AssertionError: expected [] but found [EMERGENCY]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="5.778">
    <failure type="java.lang.AssertionError" message="expected [] but found [EMERGENCY]">
      <![CDATA[java.lang.AssertionError: expected [] but found [EMERGENCY]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="5.786">
    <failure type="java.lang.AssertionError" message="expected [] but found [EMERGENCY]">
      <![CDATA[java.lang.AssertionError: expected [] but found [EMERGENCY]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="5.388">
    <failure type="java.lang.AssertionError" message="expected [] but found [EMERGENCY]">
      <![CDATA[java.lang.AssertionError: expected [] but found [EMERGENCY]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="5.561">
    <failure type="java.lang.AssertionError" message="expected [] but found [EMERGENCY]">
      <![CDATA[java.lang.AssertionError: expected [] but found [EMERGENCY]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="5.531">
    <failure type="java.lang.AssertionError" message="expected [] but found [EMERGENCY]">
      <![CDATA[java.lang.AssertionError: expected [] but found [EMERGENCY]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="5.591">
    <failure type="java.lang.AssertionError" message="expected [] but found [EMERGENCY]">
      <![CDATA[java.lang.AssertionError: expected [] but found [EMERGENCY]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="5.604">
    <failure type="java.lang.AssertionError" message="expected [] but found [EMERGENCY]">
      <![CDATA[java.lang.AssertionError: expected [] but found [EMERGENCY]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="verify_ClassificationFacility" time="5.597">
    <failure type="java.lang.AssertionError" message="expected [] but found [EMERGENCY]">
      <![CDATA[java.lang.AssertionError: expected [] but found [EMERGENCY]
at testCases.EADMF26_28_TC_02.verify_ClassificationFacility(EADMF26_28_TC_02.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_ClassificationFacility -->
  <testcase classname="testCases.EADMF26_28_TC_02" name="LogOut" time="1.55"/>
</testsuite> <!-- Test -->
